zenodo/modules/stats/templates/events/v2/file-download-v1.0.0.json

```diff
@@ -28,6 +28,7 @@
       },
       "date_detection": false,
       "numeric_detection": false,
+      "dynamic": false,
```

Comment:
Name of the doc type should be `stats-file-download` (instead of `file-download-v1.0.0`)